WebHostLib/templates/userContent.html

```diff
@@ -33,7 +49,11 @@ <h2>Your Rooms</h2>
                             <tr>
                                 <td><a href="{{ url_for("view_seed", seed=room.seed.id) }}">{{ room.seed.id|suuid }}</a></td>
                                 <td><a href="{{ url_for("host_room", room=room.id) }}">{{ room.id|suuid }}</a></td>
-                                <td>{{ room.seed.slots|length }}</td>
+                                <td>
+                                    <div title="{{ games(room.seed.slots) }}">
+                                        {{ room.seed.slots|length }}
+                                    </div>
+                                </td>
                                 <td>{{ room.creation_time.strftime("%Y-%m-%d %H:%M") }}</td>
                                 <td>{{ room.last_activity.strftime("%Y-%m-%d %H:%M") }}</td>
                                 <td><a href="{{ url_for("disown_room", room=room.id) }}">Delete next maintenance.</td>
```

Comment:
Also while you're here, these links aren't closed
```suggestion
                                <td><a href="{{ url_for("disown_room", room=room.id) }}">Delete next maintenance.</a></td>
```

Reply:
Oh, that is interesting. :D